src/views/Portal/components/ApplicationForm/Connected/index.view.tsx

```diff
@@ -1,64 +1,74 @@
 import React from "react"
 import "./index.scss"
+import { useApplication } from "components/ApplicationContext/ApplicationContext"
 import TextField from "../../FormInputs/TextField"
 import TextBox from "../../FormInputs/TextBox"
-import RadioForm from "../../FormInputs/Radio"
-// import DropDown from "../../FormInputs/DropDown"
-// import SearchBox from "../../FormInputs/SearchBox"
+import FileUpload from "../../FormInputs/FileUpload"
 
 const ConnectedPage: React.FC = () => {
-  // placeholder
-  const handleChange = () => {}
-  const errors = ""
+  const { connectedFormData, setConnectedFormData } = useApplication()!
+  const handleChange = (event: any) => {
+    const name = event.target.getAttribute("name")
+    if (Object.keys(connectedFormData).includes(name)) {
+      const copy = { ...connectedFormData, [name]: event.target.value }
+      setConnectedFormData(copy)
+    }
+  }
   return (
     <div className='connected-page'>
       <div className='connected-page__container'>
         <div className='connected-page__container--title'>Get Connected</div>
-        <div className='connected-page__container--info'>
-          Want to share your resume with our sponsors? Fill out our resume drop
-          form here.
-        </div>
-        <input type='file' />
+        <FileUpload
+          question='Want to share your resume with our sponsors? Upload your pdf resume here.'
+          errorMessage={connectedFormData.resumeErr}
+          handleChange={setConnectedFormData}
+          name='resume'
+          className='none'
+        />
         <div className='connected-page__container--body'>
           <TextField
             name='LinkedIn'
             label='LinkedIn'
```

Comment:
```suggestion
            label='linkedin'
```

These labels have to match the ones defined in [`PropBuilder`](https://github.com/CruzHacks/cruzhacks-2022-website/blob/a9108a9938c9f5256dc1c3176a4e9b9aab124f8c/src/views/Portal/utils/PropBuilder.ts#L84-L87). Otherwise the handleChange event won't work.